src/test/java/nl/ordina/jobcrawler/scrapers/YachtVacancyScraperTest.java

```diff
@@ -0,0 +1,225 @@
+package nl.ordina.jobcrawler.scrapers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.exc.MismatchedInputException;
+import nl.ordina.jobcrawler.model.Skill;
+import nl.ordina.jobcrawler.model.Vacancy;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class YachtVacancyScraperTest {
+
+    @InjectMocks
+    private YachtVacancyScraper yachtVacancyScraper;
+
+    @Mock
+    private RestTemplate restTemplateMock;
+
+    @Mock
+    private Document documentMock;
+
+    private static Document vacancyDoc;
+    private static Document removedVacancyDoc;
+
+    private static ResponseEntity<YachtVacancyResponse> jsonResponse;
+    private static ResponseEntity<YachtVacancyResponse> noDataResponse;
+
+    @BeforeClass
```

Comment:
@BeforeAll is the Junit5 equivalent of @BeforeClass. I think we are mainly using Junit5 and not Junit4.
Explained [here](https://www.baeldung.com/junit-before-beforeclass-beforeeach-beforeall)